tests/LSP7DigitalAsset/extensions/LSP7CompatibleERC20.behaviour.ts

```diff
@@ -519,44 +543,41 @@ export const shouldInitializeLikeLSP7CompatibleERC20 = (
     });
 
     it("should have set expected entries with ERC725Y.setData", async () => {
-      await expect(context.initializeTransaction).toHaveEmittedWith(
+      await expect(context.initializeTransaction).to.emit(
         context.lsp7CompatibleERC20,
-        "DataChanged",
-        [SupportedStandards.LSP4DigitalAsset.key]
-      );
+        "DataChanged"
+      ).withArgs(SupportedStandards.LSP4DigitalAsset.key);
       expect(
         await context.lsp7CompatibleERC20["getData(bytes32)"](
           SupportedStandards.LSP4DigitalAsset.key
         )
-      ).toEqual(SupportedStandards.LSP4DigitalAsset.value);
+      ).to.be.equal(SupportedStandards.LSP4DigitalAsset.value);
 
       const nameKey =
         "0xdeba1e292f8ba88238e10ab3c7f88bd4be4fac56cad5194b6ecceaf653468af1";
       const expectedNameValue = ethers.utils.hexlify(
         ethers.utils.toUtf8Bytes(context.deployParams.name)
       );
-      await expect(context.initializeTransaction).toHaveEmittedWith(
+      await expect(context.initializeTransaction).to.emit(
         context.lsp7CompatibleERC20,
-        "DataChanged",
-        [nameKey]
-      );
+        "DataChanged"
+        ).withArgs(nameKey);
       expect(
         await context.lsp7CompatibleERC20["getData(bytes32)"](nameKey)
-      ).toEqual(expectedNameValue);
+      ).to.be.equal(expectedNameValue);
 
       const symbolKey =
         "0x2f0a68ab07768e01943a599e73362a0e17a63a72e94dd2e384d2c1d4db932756";
```

Comment:
Same comment as above

```suggestion
        ERC725YKeys.LSP4["LSP4TokenSymbol"];
```